### PR TITLE
CRUD functionlity

### PR DESCRIPTION
**Pull request by Lul-Abdifan**
### CRUD
-  Remove All hardcoded items from the tasks 
-  Add crud functionality to the components
  -  Update file 
    -> When you click the Edit icon it shows a save button to save on that index
  -  Delete file
    -> Update updation of the index after deleted
    ->Add updateIndex function of supportive file 
  - Create file
-  Add render functionality
- Add a storage file that manages local storage efficiently
- Update styles file
If you have any questions or misconceptions. @Lul-Abdifan .I am here Thank you for your time and understanding                                
                    :smile: Happy Coding:computer:

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/prefer-default-export */
+
```
Comment (mwanawabangona) [inline src/index.js:2]: - [ ] `Required`
- [ ] Please enable the `disabled eslint` l because it defeats the purpose of installing the linters🥂

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @Lul-Abdifan ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbayomiOlaoye — APPROVED** [review]
Hi @Lul-Abdifan :wave:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Great implementation of reviewer's suggestions :heavy_check_mark: 
- Good CRUD implementation :heavy_check_mark: 
- Linter checks are passing :heavy_check_mark: 
- Professional ReadMe documentation :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Lul-Abdifan** [comment]
-  Required Changes Done
   -  Remove the commented code
-  Update scss file

**Pull request merged**